### PR TITLE
GCCP-8013: better information in case of not existing .npmrc

### PR DESCRIPTION
**Pull request by ivankulik-krafteq**

### CODE REVIEW

File: yais.sh [hunk 1]
```diff
@@ -144,6 +156,23 @@ do
   fi
 done
 
+if [ ! -f "$ARG_NPMRC_FILE" ]; then
+        echo "@mhie-ds:registry=https://pkgs.dev.azure.com/MHIE/_packaging/NpmMhi/npm/registry/
+; begin auth token
+//pkgs.dev.azure.com/MHIE/_packaging/NpmMhi/npm/registry/:username=NpmMhi
```
Comment (ivankulik-krafteq) [inline yais.sh:162]: Is it ok to use this name for template, or should I change it to `<your_username>`?

File: yais.sh [hunk 2]
```diff
@@ -144,6 +156,23 @@ do
...
+//pkgs.dev.azure.com/MHIE/_packaging/NpmMhi/npm/registry/:_password=<your_npm_pat>
+//pkgs.dev.azure.com/MHIE/_packaging/NpmMhi/npm/registry/:email=CiUser01@ds.mhie.com
```
Comment (ivankulik-krafteq) [inline yais.sh:164]: same question for email

**Comment by alekseysavostin** [comment]
looks good as for me

**Review by alekseysavostin — APPROVED** [review]

File: yais.sh (hunk 2 above)
Comment (ivanbenko) [reply yais.sh:164]: does it work without email?

File: yais.sh (hunk 1 above)
Comment (ivanbenko) [reply yais.sh:162]: I'd prefer to keep a placeholder `<your_username>`

File: yais.sh (hunk 2 above)
Comment (ivankulik-krafteq) [reply yais.sh:164]: npm ignores it, but requires)

File: yais.sh (hunk 1 above)
Comment (ivankulik-krafteq) [reply yais.sh:162]: I double checked, azure devops always generate file with the name of the feed. So I assume it is ok to use it like this.

**Pull request merged**